### ISSUE TITLE
Multisig tree - presentation for multilevel multisig

### ISSUE THREAD

**Issue by cryptoBeliever** [symbol/desktop-wallet#1849]
**Steps**:
1. Create multilevel multisig. For example:
```
TBSGQZ267VR2NBV6BOWEVBO6Y4KXSF3VPXGPAVA (mmm)
                TBBHSG22BSJ3JULOQCBSIHJEYXAYS6FDBXRFZVI (val1)
                                  TABX4JT3I5UTIHPQDDGTBNHP462TYHOX2BU5LRA (val3)
                                  TD65L372LP2M5QJ53TSA2PCRQ7AJMEVPLDPAXXY (val2)
                TCSS242U7AYOHHBWFMAMGQW7MH5B7NTO7ZT3FBQ (core1)
                                 TCBDXB6CDUWWXZDMGOIQZ7FJSB5TF3PHNKF2C2I (core2)
```
2. Go to account mmm (root) and open multisig tree. Result: We can see the entire tree. That's correct.

![image](https://user-images.githubusercontent.com/17684281/154963934-86ed55c7-4572-45ec-8ece-b2b9169e4185.png)

3. Go to account val1 (second-level cosigner). Result: We can't see the entire tree. Accounts core1 and core2 are not visible:

![image](https://user-images.githubusercontent.com/17684281/154963263-d2dcfb5c-ffcd-4e74-a0fd-4127302b1e17.png)

**Why is that?** 
It's because when we call tree for TBBHSG22BSJ3JULOQCBSIHJEYXAYS6FDBXRFZVI we can't see part of the tree (core1, core):
https://201-joey-dual.symboltest.net:3001/account/TBBHSG22BSJ3JULOQCBSIHJEYXAYS6FDBXRFZVI/multisig/graph

A similar issue is when we call for accounts val2 and val3, but for them we not presenting tree so it's not so visible.

**How to fix that?**
I think the easier solution on the wallet side would be to do two calls to /multisig/graph first as is now to fetch root multisig. And the second one is for root address to fetch the entire tree.

**Comment by cryptoBeliever** [comment]
Fixed in https://github.com/symbol/desktop-wallet/pull/1847

**Comment by cryptoBeliever** [comment]
Released in 1.0.11
